solutions/platinum/cses-1742.mdx

```diff
@@ -0,0 +1,270 @@
+---
+id: cses-1742
+source: CSES
+title: Robot Path
+author: Mihnea Brebenel
+---
+
+## Solution 1 - Using Segment Tree
```

Comment:
I think your solution doesn't completely explain to an outside reader how to solve the problem, so I think you should rewrite this also using a more formal tone and better grammar (Use Grammarly).